docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -5262,7 +5262,7 @@ List tasks
   - `id=<task id>`
   - `name=<task name>`
   - `service=<service name>`
-  - `node=<node id>`
+  - `node=<node id or name>`
```

Comment:
Can you update the 1.24 documentation as well? Since this is already supported in 1.12.1, that corresponds with API 1.24